kythe/cxx/common/indexing/KytheGraphRecorder.cc

```diff
@@ -31,8 +31,7 @@ static const std::string* const kNodeKindSpellings[] = {
     new std::string("interface"),  new std::string("package"),
     new std::string("tsigma"),     new std::string("doc"),
     new std::string("tbuiltin"),   new std::string("meta"),
-    new std::string("diagnostic"),
-};
+    new std::string("diagnostic"), new std::string("clang/usr")};
```

Comment:
Is the } on the same line what clang format wanted?

Reply:
Yes.